exercises/rna-transcription/example/RNA.ceylon

```diff
@@ -0,0 +1,13 @@
+class Error() {}
+
+String|Error transcription(String dna) {
+  return dna.fold("" of String|Error)((String|Error r,
+                                       Character    c) =>   if(is Error r)
+                                                          then(r)
+                                                          else(switch(c)
```

Comment:
If the parentheses around the entire `switch` were to be removed, is it still valid Ceylon? If it is still valid, shall they be removed? I don't really have ground for this request other than "I like to avoid unnecessary parentheses)